### PR TITLE
doc(INSTALLpc): Updated Ruby config.h instructions

### PR DESCRIPTION
**Pull request by dertuxmalwieder**
RubyInstaller will want different directories. Updated.

### CODE REVIEW

File: src/INSTALLpc.txt [hunk 1]
```diff
@@ -746,11 +746,11 @@ config.h and Ruby's DLL name. Here are the steps for working around them:
 
     For 32-bit version:
 
-      xcopy /s .ext\include C:\Ruby24\include\ruby-2.4.0
+      xcopy /s .ext\include\i386-mswin32_140\config.h C:\Ruby24\include\ruby-2.4.0\ruby
 
     For 64-bit version:
 
-      xcopy /s .ext\include C:\Ruby24-x64\include\ruby-2.4.0
+      xcopy /s .ext\include\x64-mswin64_140 C:\Ruby24-x64\include\ruby-2.4.0\ruby
```
Comment (k-takata) [inline src/INSTALLpc.txt:753]: `config.h` should be placed at `C:\RubyXY\include\ruby-X.Y.0\i386-mswin32_140\config.h` (32-bit) or `C:\RubyXY-x64\include\ruby-X.Y.0\x64-mswin64_140\config.h` (64-bit), not at `C:\RubyXY{,-x64}\include\ruby-X.Y.0\ruby\config.h`.
The original commands correctly copy them.

Comment (dertuxmalwieder) [reply src/INSTALLpc.txt:753]: That leads - at least on my machine ... - to an error that config.h could not be found. The updated commands fixes that for me.

Comment (k-takata) [reply src/INSTALLpc.txt:753]: Some of the build options might not be correctly set? (E.g. `RUBY_API_VER_LONG`, `RUBY_PLATFORM`)

Comment (dertuxmalwieder) [reply src/INSTALLpc.txt:753]: The problem is that Ruby's own `internal/config.h` (used by most Ruby header files) explicitly requires a `ruby/config.h`...

--> https://github.com/ruby/ruby/blob/5124f9ac7513eb590c37717337c430cb93caa151/include/ruby/internal/config.h#L22

Comment (k-takata) [reply src/INSTALLpc.txt:753]: > `config.h` should be placed at `C:\RubyXY\include\ruby-X.Y.0\i386-mswin32_140\config.h` (32-bit) or `C:\RubyXY-x64\include\ruby-X.Y.0\x64-mswin64_140\config.h` (64-bit),

Sorry, the correct paths are:
32-bit: `C:\RubyXY\include\ruby-X.Y.0\i386-mswin32_140\ruby\config.h`
64-bit: `C:\RubyXY-x64\include\ruby-X.Y.0\x64-mswin64_140\ruby\config.h`

`C:\RubyXY\include\ruby-X.Y.0\i386-mswin32_140` or `C:\RubyXY-x64\include\ruby-X.Y.0\x64-mswin64_140` is specified as an include directory by the `-I` option in the makefile.
So, the compiler should be able to find `ruby/config.h`.

Comment (dertuxmalwieder) [reply src/INSTALLpc.txt:753]: Indeed, it does. Turns out it was a configuration mistake on my side. 🤦‍♂️
Sorry.

**Pull request closed without merging**